README.md

```diff
@@ -16,16 +16,28 @@ The Contentful Swift SDK hasn't reached 1.0, yet, but provides a much nicer API
 
 ## Usage
 
-
 ```swift
+import Dispatch
+import Contentful
+
 let client = Client(spaceIdentifier: "cfexampleapi", accessToken: "b4c0n73n7fu1")
-client.fetchEntry("nyancat") { (result) in
+
+DispatchQueue.global().sync {
+  let group = DispatchGroup()
+  group.enter()
```

Comment:
lets remove this dispatch stuffs, sync stuff just isn't as relevant in the iOS world